plutus-core/untyped-plutus-core/test/Transform/Simplify.hs

```diff
@@ -272,38 +287,50 @@ forceDelayMultiApply = runQuote $ do
       term =
         Force () $
           mkIterAppNoAnn
-          ( LamAbs () x1 $ LamAbs () x2 $ LamAbs () x3 $ LamAbs () f $
-              Delay () $ mkIterAppNoAnn (Var () f) [Var () x1, Var () x2, Var () x3]
-          )
-          [one, two, three, Var () funcVar]
+            ( LamAbs () x1 $
+                LamAbs () x2 $
+                  LamAbs () x3 $
+                    LamAbs () f $
+                      Delay () $
+                        mkIterAppNoAnn (Var () f) [Var () x1, Var () x2, Var () x3]
```

Comment:
Unfortunately this style doesn't work well with `fourmolu` (and other formatters too). 
I'd have to give up on using the formatter which I personally not ready to do.

Reply:
[Your](https://github.com/IntersectMBO/plutus/pull/5761) own experiments with `fourmolu` suggest otherwise:

![Screenshot from 2024-05-08 17-56-06](https://github.com/IntersectMBO/plutus/assets/10480926/5151ff9c-df6d-4e09-9890-6cb5d076cee3)

Why was it fine before, but not now? I only agreed to tolerate fourmolu because it screws up a somewhat small portion of my code. If it's going to screw up every Plutus Core AST I've ever created, then I'm simply not agreeing to that.

Reply:
https://github.com/fourmolu/fourmolu/issues/405